frontend/src/components/ui/chat/ChatBlock.js

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react"
+
+import TextField from "@mui/material/TextField";
+import Divider from "@mui/material/Divider";
+
+import ChatList from "./ChatList";
+import Title from "./Title";
+import style from "./ChatBlock.module.css";
+
+const DUMMY_DATA = [
```

Comment:
Good work for implementing dummy data here, it helps with connecting to backend later. Thanks!